.github/workflows/ci.yml

```diff
@@ -29,10 +30,21 @@ jobs:
       - uses: actions/setup-python@v5
         with:
           python-version: ${{ matrix.python }}
-      # Using fixed Poetry version until
-      # https://github.com/python-poetry/poetry/pull/7694 is fixed
-      - run: python -m pip install --upgrade wheel "poetry==1.4.0" poethepoet
-      - run: poetry install --with pydantic_converter --with dsl --with encryption --with trio_async
+      - run: uv tool install poethepoet
+      - run: uv sync --group=dsl --group=encryption --group=trio-async --python ${{ matrix.python }}
+      - name: Check python version
```

Comment:
Not sure we really need this Python version checker code (or the `--python` on the `uv sync`)

Reply:
Oops yes, removed.